src/CommunityVoices/App/Api/View/DisplayError.php

```diff
@@ -23,4 +23,14 @@ protected function getError($request)
 
         return $response;
     }
+    protected function getAllErrors($request)
+    {
+        $errors = explode('[]',str_replace('\n','',file_get_contents('/var/www/html/log/access.log')));
+        foreach($errors as &$item) {
+            $item = array("item" => $item);
```

Comment:
Mostly an idea for the future, but noting the format of these log files is `[time] message`, it could be cool to return two fields in the array: the time (turned into a Unix timestamp) and the message.  Then, we would become able to filter messages by timestamp, which would help *a ton* toward finding the specific error from someone.

Also, very cool usage of a pointer in PHP.  I generally don't do that, so fun to see.